app/src/main/java/com/github/h3lp3rs/h3lp/signin/SignInActivity.kt

```diff
@@ -57,14 +68,18 @@ class SignInActivity : AppCompatActivity() {
         findViewById<ImageButton>(R.id.signInButton).setOnClickListener{
             launchSignIn()
         }
+        // Sign in local storage doesn't need online sync
+        Storages.SIGN_IN.setOnlineSync(false)
+        // Check if the user is already signed in
+        offlineCheckIfSignedIn()
     }
 
     /**
      * Initialize client and launch the sign in request
      */
     private fun launchSignIn(){
         signInClient = SignIn.get()
-        checkIfSignedIn()
+        //checkIfSignedIn()
```

Comment:
is that still necessary? If not we can just remove it

Reply:
Yes just forgotten oups 😬